### PR TITLE
Escaping fixes

### PR DESCRIPTION
**Pull request by jeremydstone**
This PR fixes several escaping issues/warnings. Manually tested each, proved the code I changed was getting hit in each case, the page still worked and no errors or warnings in the console.

### CODE REVIEW

File: dashboard/app/views/gallery_activities/index.html.haml [hunk 1]
```diff
@@ -19,7 +19,7 @@
   .row
     .span2
     .span8.infobox
-      = t('gallery.activity_count', days: @days, count: number_with_delimiter(GalleryActivity.pseudocount)).html_safe
+      = t('gallery.activity_count', days: @days, count: number_with_delimiter(brakeman_no_warn(GalleryActivity.pseudocount))).html_safe
```
Comment (jeremydstone) [inline dashboard/app/views/gallery_activities/index.html.haml:22]: No bug, fixes warning - brakeman doesn't know if `GalleryActivity.pseudocount` is safe to directly render. (It is, so use `brakeman_no_warn`.)

File: dashboard/app/views/layouts/_header.html.haml [hunk 2]
```diff
@@ -130,7 +130,7 @@
   - script_data = script.summarize_header
   - stage_data = (@stage || script_level.stage).summarize
   -# If caching, we don't know the sign in state, and should be nil
-  - signed_in = @public_caching ? 'null' : !current_user.nil?
+  - signed_in = @public_caching ? 'null' : !brakeman_no_warn(current_user).nil?
```
Comment (jeremydstone) [inline dashboard/app/views/layouts/_header.html.haml:133]: No bug, fixes brakeman warning

File: dashboard/app/views/levels/editors/_md_instructions.haml [hunk 3]
```diff
@@ -53,10 +53,10 @@
 
 :javascript
   var mdEditor = levelbuilder.initializeCodeMirror('level_long_instructions', 'markdown', function (editor, change) {
-    localStorage.setItem('markdown_' + '#{@level.id || params[:type]}', editor.getValue());
+    localStorage.setItem('markdown_' + '#{@level.id || html_escape(params[:type])}', editor.getValue());
```
Comment (jeremydstone) [inline dashboard/app/views/levels/editors/_md_instructions.haml:56]: `type` is a URL param, so needs to be escaped. Tested with injecting various values before & after.

**Comment by breville** [comment]
I'm not familiar with `brakeman`.  How is it run in our org?  Just manually every so often?

**Comment by jeremydstone** [comment]
Will discuss w/you offline...

**Pull request merged**